chapters/chap03-sec02.tex

```diff
@@ -124,7 +124,7 @@ \subsection{An efficient instantiation of \mimc{} primitives}\label{instantiatio
 \[
     \constraints = \rounds \cdot \mults + 1
 \]
-where $\rounds = \lceil \frac{\log_2 \rCURVE}{\log_2 \exponent{}} \rceil$, $\mults$ is the number of multiplications required for exponentiation and the additional constraint (corresponding to $+1$ in the above formula) is a result of the final message and key addition. Note that for $\exponent = 2^t - 1$ we have $\mults = 2 \cdot t - 2$ using the \emph{square-and-multiply} algorithm~\cite{menezes1996handbook}.
+where $\rounds = \lceil \frac{\log_2 \rCURVE}{\log_2 \exponent{}} \rceil$, $\mults$ is the number of multiplications required for exponentiation and the additional constraint (corresponding to $+1$ in the above formula) is a result of the final message and key addition. Note that for $\exponent = 2^t - 1$ we have $\mults = 2 \cdot t - 2$, using the \emph{square-and-multiply} algorithm~\cite{menezes1996handbook}, and for $\exponent = 2^t + 1$ we have $mults = t + 1$.
```

Comment:
REMINDER: The statement above:

> Note that small exponents $\exponent{}$ result in fewer constraints in the arithmetic circuit ...

needs to be reworded before this is merged (assuming there is no problem with 2^t+1).